src/funcs/getDbSchemaStructures.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+const getSchemas = require('../queries/getSchemas');
+const getTablesInSchema = require('../queries/getTablesInSchema');
+const getTableStructure = require('../queries/getTableStructure');
+const getConstraints = require('../queries/getConstraints');
+
+const getPrimaryKey = (schema, tableName, constraints) => {
+  if (!constraints) return undefined;
+  return constraints.find(({ constraintType, fromSchema, fromTable }) => {
+    return fromSchema === schema &&
+      fromTable === tableName &&
+      constraintType === "PRIMARY KEY";
```

Comment:
Should there be single quotes instead of double quotes around the PRIMARY KEY?

Reply:
will fix